docs/sources/panels-visualizations/configure-thresholds/index.md

```diff
@@ -20,66 +20,104 @@ weight: 100
+| As lines                             | {{< figure max-width="500px" class="w-100p" link-class="w-fit mx-auto d-flex flex-direction-column" src="screenshot-thresholds-lines-v10.4.png" alt="Visualization with threshold as a line" >}} |
+| As lines (dashed)                    | {{< figure max-width="500px" src="screenshot-thresholds-dashed-lines-v10.4.png" alt="Visualization with threshold as a dashed line" >}}                                                          |
+| As filled regions                    | {{< figure max-width="500px" src="screenshot-thresholds-regions-v10.4.png" alt="Visualization with threshold as a region" >}}                                                                    |
+| As filled regions and lines          | {{< figure max-width="500px" src="screenshot-thresholds-lines-regions-v10.4.png" alt="Visualization with threshold as a region and line" >}}                                                     |
+| As filled regions and lines (dashed) | {{< figure max-width="500px" src="screenshot-thresholds-dashed-lines-regions-v10.4.png" alt="Visualization with threshold as a region and dashed line" >}}                                       |
 
-To add a threshold:
+## Add a threshold
 
-1. Edit the panel to which you want to add a threshold.
+You can add as many thresholds to a visualization as you want. Grafana automatically sorts thresholds values from highest to lowest.
 
-1. In the options side pane, locate the **Thresholds** section and click **+ Add threshold**.
+1. Navigate to the panel you want to update.
+1. Hover over any part of the panel you want to work on to display the menu on the top right corner.
+1. Click the menu and select **Edit**.
+1. Scroll to the **Thresholds** section of the panel edit pane.
+1. Click **+ Add threshold**.
 
-1. Select a threshold color, number, and mode.
-   Threshold mode applies to all thresholds on this panel.
+   A new threshold 10 units higher than the default threshold is added.
 
-1. For a time-series panel, select a **Show thresholds** option.
+1. Enter a new threshold value or use the up and down arrows at the right side of the field to increase or decrease the value incrementally.
+1. Click the colored circle to the left of the threshold value to open the color picker, where you can update the threshold color.
+1. Under **Thresholds mode**, select either **Absolute** or **Percentage**.
+1. Under **Show thresholds**, set how the threshold is displayed or turn it off.
 
-Delete a threshold when it is no longer needed. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
+Delete a threshold when it's no longer needed. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
```

Comment:
@nmarrs - What does line 113 mean? Is this if you've got a repeating panel or something? Or is this a reference to old functionality that doesn't exist anymore?

Reply:
hm that wording is a bit strange. AFAIK thresholds only apply to a given panel (and are saved as part of a panel's config) and are therefore dont have a dashboard scope (i.e. impact multiple viz panels). We probably can either 1) remove this entire section or 2) modify second sentence to say something like `When you delete a threshold, your metrics will be styled according to the remaining thresholds` or something like that 🤷

Reply:
I'm just gonna delete that entire line and go with just what's on line 112.